lib/srv/exec.go

```diff
@@ -137,9 +137,11 @@ func prepInteractiveCommand(ctx *ctx) (*exec.Cmd, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	// this configures shell to run in 'login' mode
+	// this configures shell to run in 'login' mode. from the bash man page:
```

Comment:
you are mentioning bash docs here, what about other shells? what does OpenSSH do in case if shell is not supported?